### DISCUSSION TITLE
Quesito su Shelly Plus PM Mini

### DISCUSSION THREAD

**Discussion by esposimo** [indomus/forum#2966] [Hardware & Firmware]
Ciao a tutti ragazzi, quesito banale (non x me) per chi è pratico di impianti elettrici

Nello Shelly Plus PM Mini (misuratore di consumi a 16A, 3000W blabla) , ci sono 4 attacchi

L & N rispettivamente fase/neutro per alimentare lo shelly e 2 "O"
O -> Fase che andrà a finire nella presa di corrente da monitorare
O -> Questa a cosa serve? Nei vari tutorial visti su youtube nessuno la utilizza (tutte prove da banco). E' per caso una fase di "cortesia" (passatemi il termine) per continuare a camminare in casa e portare la fase ad un'altra scatola?

Grazie 1000 in anticipo!

**Comment by Scialla** [comment]
Direi che sono due uscite identiche:
https://kb.shelly.cloud/knowledge-base/shelly-plus-pm-mini
